versioned/tiered/rocksdb/src/main/java/org/projectnessie/versioned/rocksdb/EntityConverter.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.projectnessie.versioned.rocksdb;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.projectnessie.versioned.store.Entity;
+
+public class EntityConverter {
+
+  /**
+   * This converts an Entity into Commit protobuf object.
+   * @param entity the entity to convert
+   * @return the object converted from the entity
+   */
+  public static ValueProtos.Commit entityToCommit(Entity entity) {
+    if (!entity.getType().equals(Entity.EntityType.MAP)) {
+      throw new UnsupportedOperationException();
+    }
+    ValueProtos.Commit.Builder builder = ValueProtos.Commit.newBuilder();
```

Comment:
Style should be that we append final to everything, unless you have a reason to do so otherwise.

Reply:
Updated.